Adaptive thresholds are useful (#3572)
quicwg/base-drafts: draft-ietf-quic-recovery.md

less than 3, to keep in line with TCP {{?RFC5681}}.

Some networks may exhibit higher degrees of reordering, causing a sender to
detect spurious losses.  Increasing reordering thresholds after losses
are spuriously detected have proven useful in TCP, such	as TCP-NCR

> {{?RFC4653}}, and are expected to be equally useful in QUIC.

Reviewer: ```suggestion
detect spurious losses. Increasing reordering thresholds after spurious
detection of losses can make the transport more robust to reordering,
as TCP-NCR {{?RFC4653}} does for TCP. Such mechanisms are
expected to make QUIC more robust to reordering as well.
```

Reviewer: never mind this -- I like @martinthomson's suggestion better.

less than 3, to keep in line with TCP {{?RFC5681}}.

Some networks may exhibit higher degrees of reordering, causing a sender to
detect spurious losses.  Increasing reordering thresholds after losses

> are spuriously detected have proven useful in TCP, such	as TCP-NCR

Reviewer 2: ```suggestion
are spuriously detected have proven useful in TCP, such as TCP-NCR
```

> detect spurious losses.  Increasing reordering thresholds after losses
> are spuriously detected have proven useful in TCP, such as TCP-NCR
> {{?RFC4653}}, and are expected to be equally useful in QUIC.

Reviewer 2: The "such as" is injected at an odd point here, which seems to break up the flow.

```suggestion
detect spurious losses.  Increasing reordering thresholds after losses, as
recommended in TCP-NCR {{?RFC4653}}, have proven to be useful in TCP and are
expected to be equally useful in QUIC.
```
